tests/integration/DisplayTest.php

```diff
@@ -137,15 +114,13 @@ public function testScreenshot()
         // Create a Display in the system
         $hardwareId = Random::generateString(12, 'phpunit');
         $xmrChannel = Random::generateString(50);
-
         // This is a dummy pubKey and isn't used by anything important
         $xmrPubkey = '-----BEGIN PUBLIC KEY-----
-MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDmdnXL4gGg3yJfmqVkU1xsGSQI
-3b6YaeAKtWuuknIF1XAHAHtl3vNhQN+SmqcNPOydhK38OOfrdb09gX7OxyDh4+JZ
-inxW8YFkqU0zTqWaD+WcOM68wTQ9FCOEqIrbwWxLQzdjSS1euizKy+2GcFXRKoGM
-pbBhRgkIdydXoZZdjQIDAQAB
------END PUBLIC KEY-----';
-
+        MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDmdnXL4gGg3yJfmqVkU1xsGSQI
```

Comment:
you've got some spaces in here that will break the public key - I think this is why the test fails